integtest/src/main/java/org/corfudb/universe/cluster/process/ProcessCluster.java

```diff
@@ -0,0 +1,29 @@
+package org.corfudb.universe.cluster.process;
+
+import com.google.common.collect.ImmutableList;
+import org.corfudb.universe.cluster.Cluster;
+import org.corfudb.universe.cluster.ClusterException;
+import org.corfudb.universe.service.Service;
+
+public class ProcessCluster implements Cluster {
+
+    @Override
+    public Cluster deploy() throws ClusterException {
+        throw new UnsupportedOperationException("Not implemented");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Not implemented" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)